source/javascripts/meetups.js

```diff
@@ -44,37 +64,35 @@
   locations.forEach( generateMarkerData );
 
   handler.buildMap(mapOptions, function() {
```

Comment:
Closing the loop here: @alexclarkofficial and @solberg talked on the other PR (https://github.com/sandraor/website/pull/10/files#r22864632) and @solberg is going to remove the conflicting code from his PR :)